File: src-tauri/src/mapper/group_message.rs
use sqlx::{types::JsonValue, FromRow};

#[derive(Clone, Debug, FromRow)]
pub struct GroupMessage {
    pub from_id: u32,
    pub to_id: u32,
    pub message_id: u32,
    pub date: String,
    pub data: String,
    pub json: JsonValue,
    pub file_path: String,
}

File: src/script/mapper/GroupMessage.ts
import {integer, sqliteTable, text} from "drizzle-orm/sqlite-core";

export const table = sqliteTable(
    "GroupMessage",
    {
        groupName: text("groupName").primaryKey(),
        groupId: integer("groupId").primaryKey(),
        fromId: integer("fromId"),
        toId: integer("toId"),
        messageId: integer("messageId").primaryKey(),
        date: text("date"),
        data: text("data"),
        json: text("json"),
        file_path: text("file_path"),
    }
);
